models/vrp.rb

```diff
@@ -191,6 +191,7 @@ def self.check_consistency(hash)
     def self.expand_data(vrp)
       vrp.add_sticky_vehicle_if_routes_and_partitions
       vrp.adapt_relations_between_shipments
+      Interpreters::MultiTrips.new.expand(vrp)
```

Comment:
```suggestion
```

Reply:
May be in the future we can move this logic out of `Interpreters::MultiTrips` into vrp` class because it is just transformation of the input. However while this logic is in the class ` Interpreters::MultiTrips`, I think it is better to keep it in the old location (unless there is a specific reason to not to) because all other expands are inside solve at the moment.